flexget/plugins/services/kodi_library.py

```diff
@@ -0,0 +1,65 @@
+from __future__ import unicode_literals, division, absolute_import
+import logging
+import json
+
+from flexget import plugin
+from flexget.event import event
+from flexget.utils.requests import RequestException
+
+log = logging.getLogger('kodi_library')
+
+JSON_URI = '/jsonrpc'
+
+
+class KodiLibrary(object):
+    schema = {
+        'type': 'object',
+        'properties': {
+            'action': {'type': 'string', 'enum': ['clean', 'scan']},
+            'category': {'type': 'string', 'enum': ['audio', 'video']},
+            'url': {'type': 'string', 'format': 'url'},
+            'port': {'type': 'integer'},
+            'username': {'type': 'string'},
+            'password': {'type': 'string'},
+            'only_on_accepted': {'type': 'boolean', 'default': True}
+        },
+        'required': ['url', 'action', 'category'],
+        'additionalProperties': False,
+    }
+
+    @plugin.priority(-255)
+    def on_task_exit(self, task, config):
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Method could be a function](https://www.codacy.com/app/kristen/Flexget/file/2167245457/issues/source?bid=3161667&fileBranchId=3161671#l31)